src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1546,6 +1555,12 @@ ucs_status_t uct_dc_mlx5_ep_check_fc(uct_dc_mlx5_iface_t *iface,
         goto out;
     }
 
+    if (fc_hash_empty) {
```

Comment:
can we check callback id instead of fc_hash_empty?

Reply:
fixed